app/models/user_answer.rb

```diff
@@ -0,0 +1,5 @@
+class UserAnswer < ActiveRecord::Base
+  validates :user_id, :answer_id, presence: true
```

Comment:
Generally validate the association records themselves not their IDs/foreign keys. Validating foreign keys allows for the breakage of referential integrity. Whaaaaaa? That means you could have IDs in here (they are "present") that don't point to any user or answer records.

If you `validates :user, :answer, presence: true` you will gain referential integrity on these records.